doc/nrf/protocols/bt/bt_mesh/configuring.rst

```diff
@@ -190,15 +192,40 @@ In a worst case scenario, the store time can be up to several minutes.
 This can for example happen when storing a large size replay protection list.
 It is recommended to configure the settings subsystem's internal caches to improve the performance.
 
-* The Settings NVS name cache reduces the number of search loops of internal parameter identifiers, keeping them in memory.
+The Settings NVS name cache reduces the number of search loops of internal parameter identifiers, keeping them in memory.
+
+* :kconfig:option:`CONFIG_SETTINGS_NVS_NAME_CACHE`
+
+The size of the Settings NVS name cache, :kconfig:option:`CONFIG_SETTINGS_NVS_NAME_CACHE_SIZE`, is recommended to be at least equal to the number of setting entries the device is expected to store. These entries may include entries in the RPL, whose size is given by :kconfig:option:`CONFIG_BT_MESH_CRPL`, :ref:`configuration values <ug_bt_mesh_configuring_optional_features>`, application- and network keys, application specific states etc. Using :ref:`bluetooth_mesh_sensor_server` as an example configured according to the sample's :ref:`configuration guide <bluetooth_mesh_sensor_server_conf_models>`, this has the following possible entries:
+
+* RPL entries. Since the default Networked Lighting Control (NLC) config :kconfig:option:`CONFIG_BT_MESH_NLC_PERF_DEFAULT` is used, the RPL size is 32.
+* Application key(s). Assume three are used.
+* Bound application keys. Each of the three Sensor Servers and Sensor Setup Servers has one bound application key.
+* Network key(s). Assume only one key is used.
+* Model subscription(s). Each of the three Sensor Servers and Sensor Setup Servers models subscribes to a group address.
+* Model publication information. Each of the three Sensor Server models publishes to a group address.
+* Virtual address(es). Assume these are not used.
+* Label UUIDs for virtual addressing. Assume these are not used.
+* Configuration values
+* Heartbeat publication information
+* IV index
+* Network information
+* Sequence number
+* Model data. Each of the Three Sensor Server models stores the following data:
+
+  * Minimum interval
+  * Delta thresholds
+  * Fast period divisor
+  * Fast cadence range
 
-  * :kconfig:option:`CONFIG_SETTINGS_NVS_NAME_CACHE`.
+* The following values are stored in the sample:
 
-* The size of the Settings NVS name cache is recommended to be set to the maximum number of devices the configured device communicates with.
-  For example, if a device communicates with 255 other devices, it is worth setting the cache to minimum 255.
+  * Temperature range
+  * Presence sensor motion threshold
+  * Ambient light level gain
```

Comment:
Btw, the sample documentation is wrong because it doesn't say about the ambient light level gain and the presence threshold: https://github.com/nrfconnect/sdk-nrf/blob/e661da007b6d82ea7be70f10e92d8ffc9e8c3af1/samples/bluetooth/mesh/sensor_server/README.rst?plain=1#L116

Could you fix it as well?

Reply:
Added separate commit for this